Explorer/Assets/DCL/PluginSystem/Global/ExplorePanelPlugin.cs

```diff
@@ -100,7 +108,6 @@ public void Dispose()
         public void InjectToWorld(ref ArchSystemsWorldBuilder<Arch.Core.World> builder, in GlobalPluginArguments arguments)
         {
             backpackController.InjectToWorld(ref builder, arguments.PlayerEntity);
-
         }
 
         public class ExplorePanelSettings : IDCLPluginSettings

```

Comment:
Let's not forget that we can use a class derived from `ComponentReference<T>` to reference components from Addressables and load directly without `GetComponent`